packages/turf-invariant/index.js

```diff
@@ -189,10 +191,36 @@ function getGeom(geojson) {
  * var geom = turf.getGeomType(point)
  * //="Point"
  */
-function getGeomType(geojson) {
-    if (!geojson) throw new Error('geojson is required');
-    var geom = getGeom(geojson);
-    if (geom) return geom.type;
+function getGeomType(geojson, name) {
+    console.warn('getGeomType will be deprecated in v5.0.0 in favor of getType');
```

Comment:
@stebogit added deprecated message to this method, I don't see any issues simply wrapping it with `getType` at the moment, the output is practically the same with the exception of raised errors.